go.mod

```diff
@@ -0,0 +1,3 @@
+module example.com/trainee-sort
```

Comment:
Why "example.com"?

Reply:
because it stands on the official go website i followed their "tutorial" because I didn't work with modules yet

Reply:
OK... wherever you read example.com, 192.0.2., 2001:db8::, ... they're placeholders.

What do you think you should replace that placeholder with?